vars/dataMigrationReport.groovy

```diff
@@ -0,0 +1,162 @@
+#!groovy
+import groovy.json.*
+import groovy.xml.MarkupBuilder
+import java.util.concurrent.*
+import java.util.Date
+
+def getESLogs(cluster, indexPattern, startDate) {
+    def elasticRequestBody = """
```

Comment:
Maybe better move this to resources as tpl for example and then use template engine to render this json?

Reply:
Fixed, add json.template file